libs/react-helpers/src/lib/grid/date-range-filter.tsx

```diff
@@ -84,16 +85,18 @@ export const DateRangeFilter = forwardRef((props: IFilterParams, ref) => {
   };
 
   useEffect(() => {
-    props.filterChangedCallback();
-  }, [value]); //eslint-disable-line react-hooks/exhaustive-deps
+    if (props && props.filterChangedCallback) {
```

Comment:
suggestion (non blocking): call this directly in the onChange callback? I see no value in using useEffect here

Reply:
this has been taken from the `ag-grid` documentation example. I can easily remove it and put it in onChange, but there are some errors happening on HMR and dev server that prevent me from properly testing how removing this would work.

Reply:
updated.

Reply:
actually, I need to put this back in the useEffect. if this is called only onChange it will break the reset functionality. filterChangedCallback should be called also when props get updated (not only value)

Reply:
we are just following ag-grid documentation example